content/posts/0.21-release.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Pyodide 0.21.0 release"
+date: 2022-08-02
+tags: ["release", "announcement"]
+author: "Hood Chatham on behalf of the Pyodide team"
+showToc: true
+TocOpen: false
+hidemeta: false
+comments: false
+# description: ""
+disableHLJS: true # to disable highlightjs
+disableShare: false
+hideSummary: false
+searchHidden: true
+ShowReadingTime: true
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+---
+
+Pyodide 0.21 is a major release focused on improvements to the systems for
+building and loading packages. We added support for out of tree package builds
+which should soon make it much easier for people to build binary packages that
+we do not include in our distribution. We added support for Rust packages and
+made major progress in supporting CMake packages. We also added 34 new packages
+including a large number of popular packages, for example bitarray,
+opencv-python, shapely, and xgboost.
+
+We moved from Emscripten v2.0.27 to Emscripten v3.1.14 which includes almost all
+of the patches we need (we still apply two patches). There were also many minor
+improvements and bugfixes to the Pyodide module and type conversions.
+
+
+
+For the complete list of changes see the
+[changelog](https://pyodide.org/en/stable/project/changelog.html#version-0-21-0).
+
+## Out of tree binary package builds for Pyodide
+
+The most exciting new feature in Pyodide 0.21.0 is that it adds experimental
+support for building binary packages for Pyodide "out of tree".
```

Comment:
```suggestion
support for building binary packages as wheels for Pyodide "out of tree", similarly to how standard Python wheels are built.
```

We need to emphasize that we are just building wheels, with a workflow close to the usual. "out of tree" doesn't really mean much for an average Python user.

BTW do we have docs for it? https://pyodide.org/en/stable/development/new-packages.html makes no mention of it. Maybe we should make this feature more prominent.